docs/content/latest/yugabyte-cloud/cloud-develop/create-databases.md

```diff
@@ -9,10 +9,10 @@ aliases:
   - /latest/yugabyte-cloud/create-databases/
 menu:
   latest:
-    parent: cloud-connect
+    parent: cloud-develop
     name: Create a database
```

Comment:
The naming could be changed to "Explore sample datasets" or something more specific since there is already a "create a database" under the left nav for quick-start. Change this even for the ycql page.

Reply:
Page is being commented out for now